uwblueprint/don-efficace
# Fixed routes and added Dashboard page

## Notion ticket link
Integrate Views with Navbar & Create View for Donation Dashboard (https://www.notion.so/uwblueprintexecs/Integrate-Views-with-Navbar-10710f3fb1dc809b8e4dd2132f265b09 & https://www.notion.so/uwblueprintexecs/Create-View-for-Donation-Dashboard-c7f6210e57a742c6b3a49d1deb0de83a)


## Implementation description
* Fixed navbar links and routing and created Dashboard component



## Steps to test
1. Change line 20 of PrivateRoute.tsx to !authenticatedUser and then test the routes
2. '/' should show the navbar and clicking on the links should go to the respective components



## What should reviewers focus on?
* Ensure navbar is working as expected and routes have the correct names


## Checklist
- [✅] My PR name is descriptive and in imperative tense
- [✅] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [✅] I have run the appropriate linter(s)
- [✅] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
